surface/include/pcl/surface/convex_hull.h

```diff
@@ -255,6 +263,9 @@ namespace pcl
       /* \brief z-axis - for checking valid projections. */
       const Eigen::Vector3d z_axis_;
 
+      /* \brief vector containing the point cloud indices of the convex hull points. */
+      std::vector<uint32_t> hull_indices_;
```

Comment:
I agree that ideally point indices should be stored as unsigned integers in all of PCL. But this is something we can not change now. So that being said, I see no reason to use `uint32_t` in this case. Firstly, it's not consistent with the rest of PCL. Secondly, when you fill the `hull_indices_` vector, you use indices from `indices_` vector, and inside there are good 'ol `int`s anyway. Thirdly, and most importantly, if you use a vector of `int`s , then the getter function becomes a one-liner: `hull_point_indices.indices = hull_indices_;`.